nameparser/config/titles.py

```diff
@@ -166,6 +166,7 @@
     'chef',
     'chemist',
     'chief',
+    'chief justice',
```

Comment:
I didn't realize that "Justice" was a common first name.  I don't think my "Chief Justice" string is being matched due to the prior parsing actions.  I'm not sure what qualifies as "common first name".  "Justice" is around the 580th most common first name in America.  However, I think it is probably more common than the number of judges.

Reply:
I spoke with a judge and asked him about the use of the title "Justice".  He said it was rare.  I'll undo this change, since it was based on a false assumption.

The judge expressed some dismay that titles were being used as first name.  He has encountered people with first names, such as "King" and "Queen", in his courtroom.

We might want to include the parser's bias for first names over titles.

If someone is parsing names of titled people (think UN delegations), what should they do?

Reply:
A bias for first names over titles is already a feature of the parser, and why there are no potential first names in the titles constant. First job of a name parser is to parse names, then it can optionally parse titles but not if it messes up names.

Reply:
I'll do some testing after removing "Justice" and "Chief Justice" from the list.  I might add a tests for "David Justice" and "Justice, David", the baseball player.

Reply:
I work in legal data, and I'll note that there are very few "Justices" though I suppose "justice of the peace" is a title. But in my experience, "justice" is reserved pretty much exclusively for the SCOTUS justices. You can see the way this shakes out on this page (though it doesn't discuss this topic): https://www.uscourts.gov/judges-judgeships/about-federal-judges

Reply:
Oh, sorry, meant to say that relatedly, "J." is a very common title among judges. I'm guessing it can't be added b/c it's one letter, but I thought I'd throw that out there.